<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>678</width>
    <height>599</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="5" column="0">
    <widget class="QProgressBar" name="progressBar">
     <property name="value">
      <number>0</number>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QTreeWidget" name="texList">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::ExtendedSelection</enum>
     </property>
     <column>
      <property name="text">
       <string/>
      </property>
     </column>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>50</height>
      </size>
     </property>
     <property name="title">
      <string/>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QCheckBox" name="fSRGB_CB">
          <property name="text">
           <string>Force sRGB</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="fLinear_CB">
          <property name="text">
           <string>Force Linear</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="texType_DD">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <item>
           <property name="text">
            <string>Texture</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Photometric IES</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Sphere IBL</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Hemisphere IBL</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Mirrorball IBL</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Angular IBL</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Sprite Cut-Out Map (Color Intensity)</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Sprite Cut-Out Map (Alpha)</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="parseScene_B">
       <property name="text">
        <string>Parse Scene</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pickImages_B">
       <property name="text">
        <string>Pick Images</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pickDir_B">
       <property name="text">
        <string>Pick Directory</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="6" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,0">
     <property name="spacing">
      <number>6</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QPushButton" name="convertSel_B">
       <property name="text">
        <string>Convert Selected</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="convert_B">
       <property name="text">
        <string>Convert</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QPushButton" name="clearSel_B">
       <property name="text">
        <string>Clear Selected</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="clear_B">
       <property name="text">
        <string>Clear All</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
